contracts/tunnel-consumer/src/msg.rs

```diff
@@ -0,0 +1,29 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Uint64;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub struct UpdateTunnelConfigMsg {
+    pub tunnel_id: Uint64,
+    pub port_id: String,
+    pub channel_id: String,
+}
```

Comment:
according to our new tunnel version, channel_id on the destination should cover information about tunnel_id already (from port_id on the source chain). also port_id here should be fixed as (wasm.{contract_address}). maybe we can reduce it to be just an array of channel_id?